README.md

```diff
@@ -2,15 +2,130 @@
 
 # AutoStep
 
-AutoStep is a new way of writing Gherkin-style tests, that reduces the requirement on programmers to be heavily involved in writing tests.
+AutoStep is a compiler and runner for Gherkin-style BDD tests, with a few extra language features
+on top of standard gherkin that add useful functionality.
 
-It introduces a Gherkin+ syntax that adds a number of features to the language, specifically:
+---
 
- - Specifying 'options' on tests, as well as tags.
- - The ability to define steps inside AutoStep, improving usability.
+**Status**
 
-In addition, AutoStep will come with its own editor that speeds up the test writing process.
+AutoStep is currently under development (in alpha). You can grab the CI package
+from our feedz.io package feed: https://f.feedz.io/autostep/ci/nuget/index.json. 
 
-## Architecture
+At the moment the compiler and linker is relatively stable, with work starting on test execution.
 
-  - AutoStep is built in .NET, targeting .netstandard21 for all libraries, and netcoreapp31 for all applications.
\ No newline at end of file
+Get the 'alpha' pre-release for latest develop.
+
+---
+
+# Why AutoStep
+
+Why does AutoStep exist? The team behind AutoStep has spent years testing big web applications using the existing Gherkin technologies (mostly SpecFlow) and while we are a big fan of these, we've also identified a couple of things we'd like to add. Here are a few of the useful features in AutoStep that help us with that.
```

Comment:
rather than "big web apps" can you say "enterprise-level"?

Reply:
Changed to 'enterprise-level web applications'